openedx_caliper_tracking/tests/tests.py

```diff
@@ -102,7 +137,12 @@ def test_event_is_not_called_without_settings(self, delivery_mock):
     )
     @override_settings(
         CALIPER_DELIVERY_ENDPOINT='http://localhost:3000',
-        CALIPER_DELIVERY_AUTH_TOKEN='test_auth_token',
+        CALIPER_DELIVERY_AUTH_TOKEN='test_auth_token'
```

Comment:
why change on this line ?

Reply:
trailing `,` is removed

Reply:
just for the consistency, I removed trailing commas